aiida/cmdline/commands/cmd_import.py

```diff
@@ -159,8 +205,7 @@ def cmd_import(archives, webpages, group, extras_mode_existing, extras_mode_new,
                     comment_mode=comment_mode)
             except exceptions.IncompatibleArchiveVersionError as exception:
                 echo.echo_warning('{} cannot be imported: {}'.format(archive, exception))
-                echo.echo_warning('download the archive file and run `verdi export migrate` to update it')
-                continue
+                echo.echo_critical('download the archive file and run `verdi export migrate` to update it')
```

Comment:
since these are not two independent messages, probably  better to construct the string first and then pass it to `echo_critical` directly

Reply:
I agree, I thought to do this as well, but forgot it again as quickly, unfortunately. Good catch.